targets/simple_switch_grpc/tests/utils.cpp

```diff
@@ -49,6 +49,15 @@ int get_action_id(const p4::config::P4Info &p4info,
   return 0;
 }
 
+int get_dc_id(const p4::config::P4Info &p4info,
```

Comment:
full name may be better here, even though I've been lazy in the past

Reply:
Fixed